blockchain/chain/src/store/tip_index.rs

```diff
@@ -2,18 +2,18 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use super::errors::Error;
-use blocks::{TipSetKeys, Tipset};
+use blocks::{Tipset, TipsetKeys};
 use cid::Cid;
 use clock::ChainEpoch;
 use std::collections::{hash_map::DefaultHasher, HashMap};
 use std::hash::{Hash, Hasher};
 
-/// TipSetMetadata is the type stored as the value in the TipIndex hashmap.  It contains
+/// TipsetMetadata is the type stored as the value in the TipIndex hashmap.  It contains
 /// a tipset pointing to blocks, the root cid of the chain's state after
 /// applying the messages in this tipset to it's parent state, and the cid of the receipts
 /// for these messages.
 #[derive(Clone, PartialEq, Debug)]
-pub struct TipSetMetadata {
+pub struct TipsetMetadata {
```

Comment:
```suggestion
pub struct TipsetMetaData {
```
nit: If we are already changing the name, should probably be fixed

Reply:
I think metadata is one word nowadays, so that part seems to have been correct to begin with